Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in hydra-works.gemspec
 gemspec
 
+gem 'active-fedora', github:'projecthydra/active_fedora', branch: 'master'
```

Comment:
we probably shouldn't merge this PR until these are resolved to released versions.